docs/partner-apps/bundleb2b/b2b-edition.md

```diff
@@ -0,0 +1,162 @@
+
+When done, the object will resemble the following:
+
+```jsx
+window.b3themeConfig.useText = {
+/* B3 will now use the call to action "Place Quick Order" instead of the default "Quick Order Pad" for the button that is appended to the secondary navigation menu */
+ 'nav.button.quickOrderPad': 'Place Quick Order',
+}
+```
+
+For B3 element names and default text values, see the [BundleB2B Developer Guide](https://developer.bundleb2b.net/storefront/text.html).
+
+## Overwriting and injecting JavaScript
+
+B3 has lifecycle methods for many modules that allow you to inject custom JavaScript functions at different times during page render. Each supported module has the following four global keys:
+
+```jsx
+overwrite: false,
+callback() {},
+beforeMount() {},
+mounted() {}
+```
+
+- `overwrite` is a `boolean` that specifies whether to overwrite B3 functions for that module. Default is `false`.
+- `callback()` is a `function` that will call all enclosed functions regardless of the value of `overwrite` after all other B3 code has run. Default is `noop`.
+- `beforeMount()` is a `function` that calls all enclosed functions before the module renders. Default is `noop`.
+- `mounted()` is a `function` that calls all enclosed functions after the module has rendered. Default is `noop`.
+
+To overwrite and/or inject custom functions for a supported B3 module, insert `window.b3themeConfig.useJavaScript = {}` into your theme's `assets/js/global.js` file. Within `window.b3themeConfig.useJavaScript = {}`, create a property for each of the modules you'd like to customize that uses the module name as the key and the aforementioned global keys as the value.
+
+When done, your object will resemble the following example that demonstrates the call stack of each function:
+
+```jsx
+window.b3themeConfig.useJavaScript = {
+ quickorderpad: {
+        overwrite: false,
+        callback() {
+          console.trace("quickorderpad.callback() runs after all other quickorderpad functions")
+        },
+        beforeMount() {
+     console.trace("quickorderpad.beforeMount() runs before quickorderpad mounts")        
+},
+        mounted() {
+          console.trace("quickorderpad.mounted() runs after quickorderpad mounts");
+}
+```
+
+The output to the browser console of the above example is the following:
+
+![B3 console log output](https://github.com/bigcommerce/dev-docs/blob/fc576cd09ee4f346d2668d97082a75aeff7ff468/assets/images/B3-console.png?raw=true)
+
+For a full list of available modules, see the [BundleB2B Developer Guide](https://developer.bundleb2b.net/storefront/js.html).
+
+## B3 REST API
+
+You can find instructions on how to get your B3 authentication token on the [BundleB2B Developer Guide](https://developer.bundleb2b.net/storefront/api-call.html) page.
+
+For the complete list of API endpoints, see [B3 Open API (v2) specifications](https://developer.bundleb2b.net/openapi/).
+
+## Resources
+
```

Comment:
Can get rid of all the "- BundleB2B Developer Guide" mentions to prevent redundancy